metadata/virtual_container_host.go

```diff
@@ -36,13 +37,18 @@ type VirtualContainerHostConfigSpec struct {
 	// The base config for the appliance. This includes the networks that are to be attached
 	// and disks to be mounted.
 	// Networks are keyed by interface name
-	ExecutorConfig
-
-	// The set of components to launch
-	ComponentConfig []SessionConfig
-
+	ExecutorConfig `vic:"0.1" scope:"read-only" recurse:"depth=0"`
```

Comment:
Is this `0.1` appearing throughout referring to the VIC version? Or is it the metadata schema version? At the least we should probably make this value a `const`, right?